cobra/core/Model.py

```diff
@@ -96,10 +96,13 @@ def copy(self):
         than deepcopy
         """
         new = self.__class__()
-        do_not_copy = {"metabolites", "reactions", "genes"}
+        do_not_copy = {"metabolites", "reactions", "genes", "notes",
+                       "annotation"}
         for attr in self.__dict__:
             if attr not in do_not_copy:
                 new.__dict__[attr] = self.__dict__[attr]
+        new.notes = copy(self.notes)
```

Comment:
deepcopy would probably fit better, because nested objects would also need copies, not references